lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,5 @@
+class TicTacToe
+  def printWelcome
```

Comment:
in Ruby, use `snake_case` for symbols, methods and variables.

Side note: research Ruby style Guide. Is it possible to be sure that the code we are writing is not violating existing coding conventions?

Reply:
To be sure that I'm not violating conventions, would a linter be a good idea?

Reply:
https://github.com/8thlight/learning-trails/blob/master/languages/ruby.md#ongoing-reference